javascript-web/README.md

```diff
@@ -0,0 +1,40 @@
+# Usage
+### package.json
+```
+dependencies: {
+  "@gomomento/generated-types-webtext": "$current_version",
+  "grpc-web": "1.3.1",
+  "google-protobuf": "3.21.0"
+}
+```
+
+### Basic usage shape
+```typescript
+const authToken = getClientAuthToken_never_harcode_server_api_keys();
+const cacheName = 'your_cool_cache';
+const momento = new cache.ScsClient('https://cache.cell-us-east-1-1.prod.a.momentohq.com', null, {})
```

Comment:
is this just pseudocode or does it reflect what you actually expect the user to write?

The main reason that I ask is that we currently have this abstraction, `SimpleCacheClient`, that hides the generated `ScsClient`.  In the design review for SDK configuration, I asked if we ever expected end users to construct the lower-level ones directly, because it had some implications on the choice of where we would manage the gRPC channels in the cases where we are creating more than one.  I believe we decided that we would treat these as effectively private, and users would only construct the higher-level one.

Reply:
This is what I expect someone who uses the protos to write. We will use the protos, and others can too. It's okay if people want to use the protos - but we're not offering a whole lot of help to do so. "Effectively private" is fine - but they're open source and totally fair game =)